.changeset/rare-toes-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-devops-backend': minor
```

Comment:
Is there any breaking changes? If not them this is probably a `patch`.

Reply:
Yeah, I don't think there are